DESCRIPTION

```diff
@@ -26,4 +28,5 @@ Suggests:
 Config/testthat/edition: 3
 Remotes:
     mrc-ide/orderly2,
-    reside-ic/porcelain
+    reside-ic/porcelain,
+    r-lib/gert
```

Comment:
are we using dev gert for some new feature? Can you add a version requirement in the Imports please so we know when we can remove this?  Or if CRAN gert is sufficient then drop this?

Reply:
Yep using dev gert as it includes changes to return modified time from a `git_ls` with a `ref` seehttps://github.com/r-lib/gert/pull/212 , didn't coincide with a version bump though! But added dependency on latest version number.